pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -84,12 +84,13 @@ type TransportServerListener struct {
 
 // Upstream defines an upstream.
 type Upstream struct {
-	Name        string       `json:"name"`
-	Service     string       `json:"service"`
-	Port        int          `json:"port"`
-	FailTimeout string       `json:"failTimeout"`
-	MaxFails    *int         `json:"maxFails"`
-	HealthCheck *HealthCheck `json:"healthCheck"`
+	Name           string       `json:"name"`
+	Service        string       `json:"service"`
+	Port           int          `json:"port"`
+	FailTimeout    string       `json:"failTimeout"`
+	MaxFails       *int         `json:"maxFails"`
+	MaxConnections *int         `json:"maxConnections"`
```

Comment:
could we use `maxConns` as the name of the field.
With new resources - Policies and TransportServer - we started using camelCase for field names. However, we still want to be consistent with the VS/VSR resources. For max conns, VS/VSR upstreams have `max-conns` field. So could we have a similar name here?

Reply:
I think the name is fine. I'd prefer if it was consistent within the resource.